docs/installation.md

```diff
@@ -250,11 +235,11 @@ To configure delayed termination on the NGF Pod when the deployment method is He
 1. Uninstall the NGINX Gateway Fabric:
 
    ```shell
-   kubectl delete -f deploy/manifests/nginx-gateway.yaml
+   kubectl delete -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.0.0/nginx-gateway.yaml
    ```
 
    ```shell
-   kubectl delete -f deploy/manifests/crds
+   kubectl delete -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.0.0/crds.yaml
    ```
 
 1. Uninstall the Gateway API resources from the standard channel (the CRDs and the validating webhook):
```

Comment:
Also, the [Configure Delayed Termination Using Manifests](https://github.com/nginxinc/nginx-gateway-fabric/blob/8160c3a14f7c5b9b158ffc7f07c4e0067a69f61f/docs/installation.md#configure-delayed-termination-using-manifests) section references the repo files.